t/24-myjsonrpc.t

```diff
@@ -3,6 +3,9 @@
 # Copyright 2019-2020 SUSE LLC
 # SPDX-License-Identifier: GPL-2.0-or-later
 
+BEGIN {
+    $ENV{PERL_MYJSONRPC_DEBUG} = 0;
+}
```

Comment:
We do have `t/24-myjsonrpc-debug.t` for that already.

Reply:
I set to 0 not to 1